conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -20554,3 +20554,15 @@ msgid ""
 "A new revision was created: Merge between Revision #%(r1)d and Revision "
 "#%(r2)d"
 msgstr ""
+
+#: lms/templates/widgets/cookie-consent.html
```

Comment:
If you mark strings for translation (which you properly did in the template file, using ugettext), then they will automatically get scraped and added to translation files.

See http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html.